recipes/conversation_view/time_pill/time_pill.stories.js

```diff
@@ -0,0 +1,60 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import DtRecipeTimePill from './time_pill';
+import DtRecipeTimePillMdx from './time_pill.mdx';
+import DtRecipeTimePillDefaultTemplate from './time_pill_default.story.vue';
+
+// Default Prop Values
+export const argsData = {
+};
+
+/**
+ * example prop description decorator
+ */
+
+/*
+  Controls
+  ========
+
+  Here we define any custom controls or control overrides for our components.
+
+  By default storybook will attempt to provide an appropriate control of the same name for each property in the
+  component as well as include any description provided using a prop decorator within your component (see above).
+
+  Storybook will also attempt to provide an appropriate control for each slot in the component as well as include any
+  description provided using a slot decorator within your component (see below).
+
+  <!-- @slot example slot decorator -->
+*/
+export const argTypesData = {
+};
+
+// Story Collection
+export default {
+  title: 'Recipes/Conversation View/Time Pill',
+  component: DtRecipeTimePill,
+  args: argsData,
+  argTypes: argTypesData,
+  excludeStories: /.*Data$/,
+  parameters: {
+    docs: {
+      page: DtRecipeTimePillMdx,
+    },
+  },
+};
+
+// Templates
+const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeTimePillDefaultTemplate,
+);
+
+// Stories
+export const Default = DefaultTemplate.bind({});
+
+const today = new Date();
+
+Default.args = {
+  dateTime: today.toISOString().split('T')[0],
```

Comment:
Let's set a specific date instead of using "today" as this will break the visual tests every time the day changes.

Reply:
Actually never mind it won't will it, because it only displays "today"